src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/jena/JenaUtils.kt

```diff
@@ -59,6 +59,13 @@ private fun Resource.addRegistration(org: Organization): Resource =
             .addProperty(SKOS.notation, org.organizationId)
             .addProperty(ADMS.schemaAgency, "Brønnøysundregistrene"))
 
+private fun Resource.addOrgType(orgType: String?): Resource =
+    if(orgType == null) this
```

Comment:
formattering

Reply:
kan sikkert bruke .also funksjon her

Reply:
La på whitespace :+1:

Reply:
.also vil vel føkke opp hva som refereres til som `this`?